README.rst

```diff
@@ -115,11 +115,11 @@ The :code:`n` is the PDO index (normally 1 to 4). The second form of access is f
     # network.connect(bustype='nican', channel='CAN0', bitrate=250000)
 
     # Read a variable using SDO
-    device_name = node.sdo['Manufacturer device name'].raw
-    vendor_id = node.sdo[0x1018][1].raw
+    device_name = node.sdo['Manufacturer device name'].get_raw()
+    vendor_id = node.sdo[0x1018][1].get_raw()
```

Comment:
Well, in my eyes this really kills the spirit of the Python library, it's just more reading without any more information.  The whole point of properties and their accessors is that the (average) API user should not care about where it comes from, they just want the value.  In this case, it is stored somewhere else, so must be fetched.

Of course for advanced use cases, blocking I/O functions should be clearly noted.  I'm fine with having variants that will work with async etc.  We just shouldn't burden the user with more complicated names, or warn them when using a perfectly fine (synchronous, blocking) API.